core-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -549,20 +553,6 @@
     <servlet-name>AttachmentAjaxServlet</servlet-name>
     <servlet-class>org.silverpeas.web.attachment.AjaxServlet</servlet-class>
   </servlet>
-  <!--
-  <servlet>
```

Comment:
AjaxFileUploadServlet.java should also be removed I guess.

Reply:
Done. I removed also all the upload package